src/components/PromptHeader/PromptHeader.scss

```diff
@@ -0,0 +1,7 @@
+@import '../../styles/partials/variables.scss';
+
+.prompt-header {
+  background-color:$Powder-Blue;
+  border: 2px solid black;
+  width: 100%;
+}
```

Comment:
Can you add some padding here?
![Screen Shot 2024-04-28 at 12 34 19 PM](https://github.com/makeitMVPadmin/Content/assets/146603843/1f5f2d47-3e6a-4677-b365-aafb92de0402)